### PR TITLE
Update schema.md

### PR DESCRIPTION
**Pull request by illyabra**
updated variables to change from total_discount to total_discount_dollar and adding total_discount_percentage support

### CODE REVIEW

**Review by danielpatricio — APPROVED** [review]
Clarity of this is great.

Just to recap what we said on the call - we have updated this but `total_discount` is still live for some people and we may want to update those folks to deprecate in the future.

**Comment by illyabra** [comment]
@danielpatricio  Thats correct, so `total_discount` stay as it does, im just removing the mention of it in the schema doc file. we have the additional `total_discount_dollar` to work the same way, and we have a new `total_discount_percentage` working to show the % value rounded like how shopify does it. You can take a look at the commits from Joao here before i merge anything: https://github.com/merchantinresidence/abra/pull/479

**Comment by danielpatricio** [comment]
Good to go when you are!

**Pull request merged**
